_includes/tutorials/optimize-producer-throughput/confluent/markup/dev/make-topic.adoc

```diff
@@ -0,0 +1,14 @@
+In this step we’re going to create a topic for use during this tutorial.
+Use the following command to create the topic:
+
+```
+ccloud kafka topic create topic-perf
+```
+
+This creates a topic called `topic-perf` with a default number of `6` partitions.
+A topic partition is the unit of parallelism in Kafka, and messages to different partitions can be sent in parallel by producers, written in parallel by different brokers, and read in parallel by different consumers.
+In general, a higher number of topic partitions results in higher throughput, and to maximize throughput, you want enough partitions to distribute them across the brokers in your cluster.
+Although it might seem tempting just to create topics with a very large number of partitions, there are link:https://www.confluent.io/blog/how-choose-number-topics-partitions-kafka-cluster[trade-offs] to increasing the number of partitions.
+Be sure to choose the partition count carefully based on producer throughput and consumer throughput, and benchmark performance in your environment.
```

Comment:
FYC:
```suggestion
Choose partition count carefully by benchmarking producer and consumer throughput requirements in your environment.
```

Reply:
The meaning has changed in the suggestion, one doesn't benchmark requirements...

Reply:
Hmm, one might!   

I think the current sentence has redundancies (throughput twice).   Also, IMO ", and benchmarking performance in your environment" isn't very succinct and feels redundant w/ the first half.    What performance am I to benchmark?  Throughput? that's mentioned before the comment (twice).  Up to you of course

Reply:
@rspurgeon latest commit has tweaked this sentence, WDYT?

Reply:
LGTM